src/torch_tomogram/tomogram.py

```diff
@@ -27,13 +34,95 @@ def __init__(
         self.sample_translations = torch.as_tensor(
             sample_translations, device=device
         ).float()
+        self.pixel_spacing = pixel_spacing
         self.device = device
         self._pad_factor = 2.0
+    
+    @property
+    def sample_translations_px(self) -> torch.Tensor:
+        """Get sample translations in pixels.
+        
+        If pixel_spacing was provided, converts from Angstroms to pixels.
+        Otherwise, returns the translations as-is (already in pixels).
+        """
+        if self.pixel_spacing is not None:
+            return self.sample_translations / self.pixel_spacing
+        return self.sample_translations
```

Comment:
```suggestion
        return self.sample_translations / self.pixel_spacing
```

good example of lower overall implementation complexity from removing the optional pixel spacing above 🙂